shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -967,6 +894,14 @@ - (void)updateViewportPadding {
 
 - (void)keyboardWillChangeFrame:(NSNotification*)notification {
   NSDictionary* info = [notification userInfo];
+
+  if (@available(iOS 9, *)) {
+    // Ignore keyboard notifications related to other apps.
+    if (![info[UIKeyboardIsLocalUserInfoKey] boolValue]) {
```

Comment:
I couldn't find guidance in the documentation on whether this key will always be present. The discussion in the docs seems to be around multitasking on iPad. As it stand, if the key is not present in the dictionary, the frame change will be ignored. A more defensive writing of this piece of code would be to check if the key exists and then check its bool value. If the key doesn't exist, process the frame change as normal.

Reply:
Done.  In practice it isn't just iPad, these were the very notifications that were messing us up and I was testing with iPhone.  I believe the key is always present (except iOS 8).  It doesn't hurt to make it safer.